docker-compose.yml

```diff
@@ -1,14 +1,33 @@
 ---
 version: "3.7"
+<<<<<<< HEAD
+=======
+
+# This Docker composition file is used to build and test the container
+
+secrets:
+  quote_txt:
+    file: ./src/secrets/quote.txt
+
+>>>>>>> 9cea64da4a1379cdde3c509a09fc548f0da8b034
```

Comment:
You have committed a merge conflict.  Please resolve.

Reply:
resolved with [d38e133](https://github.com/cisagov/domain-manager-ui/pull/247/commits/d38e1335f747335baa33f87d2745c9cf0cc84808)